packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Expressions/ExpressionsTest.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.IO;
+using System.Text;
+using Microsoft.Generator.CSharp.Expressions;
+using Microsoft.Generator.CSharp.Providers;
+using Microsoft.Generator.CSharp.Snippets;
+using Microsoft.Generator.CSharp.Statements;
+using Moq;
+using NUnit.Framework;
+using static Microsoft.Generator.CSharp.Snippets.Snippet;
+
+namespace Microsoft.Generator.CSharp.Tests.Expressions
+{
+    internal class ExpressionsTest
+    {
+        private readonly string _mocksFolder = "Mocks";
+        private readonly string _licenseString = "// License string";
```

Comment:
You can just pass `false` to your ToString call and avoid always needing to compare this header portion